### ISSUE TITLE
Question about API methods soon to be deprecated.

### ISSUE THREAD

**Issue by gregarican** [DavidRouyer/pipedrive-dotnet#61]
I have found this project very useful for some production integrations I've implemented since the beginning of 2020. Quick question. Checking the Pipedrive API site, I see mention of some API methods that will be eliminated March 31, 2021 (see below). Were any of these API methods present as of assembly file version 0.1.10.0 of this project?

---------------------------------------------------------

Warning!
Published: 8 April 2020

On 31 March 2021 we will remove the following endpoints:

GET /deals/find
GET /organizations/find
GET /persons/find
GET /products/find
GET /searchResults
GET /searchResults/field
In case you are using the above-mentioned API endpoints in your code or 3rd party app/integration, please make sure to read the migration guide at https://pipedrive.readme.io/docs/new-search-api-migration-guide.

**Comment by DavidRouyer** [comment]
I replaced the old ones with the new ones in this release: https://github.com/DavidRouyer/pipedrive-dotnet/releases/tag/v0.5.0 So feel free to upgrade. It's important to stay up to date because I fix the issues in the library as I encounter them in production!

**Comment by gregarican** [comment]
Sounds good! Do the new releases contain this one quick patch I did on my own local copy?
https://github.com/DavidRouyer/pipedrive-dotnet/issues/39

**Comment by DavidRouyer** [comment]
Please open a Pull Request if you want to make a contribution

**Comment by gregarican** [comment]
I can I suppose, although it's literally two lines of code. :) Thanks for your help and responsiveness. Great project!

**Comment by gregarican** [comment]
Okay, I did indeed issue a Pull Request --> https://github.com/DavidRouyer/pipedrive-dotnet/pull/62. Feel free to review the specifics. I forgot a few other quirks that I ran into. One involved some JSON timestamp fields in the JSON API response body that weren't properly mapping to DateTime? objects. So I changed them to be string data types (like the time fields). Then there were a few fields that are exposed in the API but weren't defined in the mapped class properties.
